solidity/contracts/BancorNetwork.sol

```diff
@@ -10,9 +10,14 @@ import './utility/interfaces/IContractFeatures.sol';
 import './utility/interfaces/IWhitelist.sol';
 import './token/interfaces/IEtherToken.sol';
 import './token/interfaces/ISmartToken.sol';
-import './token/interfaces/INonStandardERC20.sol';
 import './bancorx/interfaces/IBancorX.sol';
 
+// interface of older converters for backward compatibility
+contract ILegacyBancorConverter is IBancorConverter {
+    function change(IERC20Token _fromToken, IERC20Token _toToken, uint256 _amount, uint256 _minReturn) public payable returns (uint256);

```

Comment:
This function is not `payable` in converter version prior to the current one.

Reply:
removed the `payable` modifier